test/api/unit/models/build.test.js

```diff
@@ -71,11 +99,16 @@ describe('Build model', () => {
 
   describe('.completeJob(message)', () => {
     it('should mark a build errored with a message', (done) => {
-      factory.build().then(build =>
-        build.updateJobStatus({ status: 'error', message: 'this is an error' })
-      ).then((build) => {
+      factory.build({ state: 'processing', startedAt: new Date() })
```

Comment:
Might be enough logic that it's worth testing all the different state transitions